modules/ROOT/pages/mongodb/mongodb-connector-6-0-reference.adoc

```diff
@@ -0,0 +1,1442 @@
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Array of Binary
+|===
+
+=== For Configurations
+* <<config>>
+
+==== Throws
+* MONGO:DUPLICATE_ERROR
+* MONGO:CONSISTENCY_ERROR
+* MONGO:NOT_FOUND
+* MONGO:PERMISSIONS_DENIED
+* MONGO:TIMEOUT
+* MONGO:INVALID_INPUT
+* MONGO:CONNECTIVITY
+
+
+[[findFiles]]
+=== Find Files
+`<mongo:find-files>`
+
+
+Lists all files that match the given query
+
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Query a| Binary |  The query object used to list all files. Values can be {"field1": "value1"} or can contain operators { "field1": { $gte: 1, $lt: 10 } }. ||
+| Sort a| Binary |  Enables you to sort the result list. A file can only be sorted by file_id or uploadDate. |  |
+| Target Variable a| String |  The name of a variable to store the operation's output. |  |
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  #[payload] |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Array of Binary
+|===
+
+=== For Configurations
+* <<config>>
+
+==== Throws
+* MONGO:DUPLICATE_ERROR
+* MONGO:CONSISTENCY_ERROR
+* MONGO:RETRY_EXHAUSTED
+* MONGO:NOT-FOUND
+* MONGO:PERMISSIONS_DENIED
+* MONGO:TIMEOUT
+* MONGO:INVALID_INPUT
+* MONGO:CONNECTIVITY
+
+
+[[getFileContent]]
+=== Get File Content
+`<mongo:get-file-content>`
+
+Gets an InputStream to the content of the file specified by File Id.
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| File Id a| Binary |  The mandatory file ID. |  #[payload] |
+| Target Variable a| String |  The name of a variable to store the operation's output. |  |
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  #[payload] |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|*Type* a| Any
+|*Attributes Type* a| Binary
+|===
+
+=== For Configurations
+* <<config>>
+
+==== Throws
+* MONGO:DUPLICATE_ERROR
+* MONGO:CONSISTENCY_ERROR
+* MONGO:RETRY_EXHAUSTED
+* MONGO:NOT_FOUND
+* MONGO:PERMISSIONS_DENIED
+* MONGO:TIMEOUT
+* MONGO:INVALID_INPUT
+* MONGO:CONNECTIVITY
+
+
+[[insertDocument]]
+=== Insert Document
+`<mongo:insert-document>`
+
+
+Insert a document in a collection. If you don't use the _id field to specify an object, MongoDB automatically generates one.
```

Comment:
```suggestion
Insert a document in a collection. If you don't use the `_id` field to specify an object, MongoDB automatically generates one.
```

Reply:
Leaving the field as is. Backticks are not used for field names in this doc.